packages/cli/src/CredentialsHelper.ts

```diff
@@ -257,17 +267,17 @@ export class CredentialsHelper extends ICredentialsHelper {
 			throw new Error(`Credential "${nodeCredential.name}" of type "${type}" has no ID.`);
 		}
 
-		const credential = userId
-			? await Db.collections.SharedCredentials.findOneOrFail({
-					relations: ['credentials'],
-					where: { credentials: { id: nodeCredential.id, type }, userId },
-			  }).then((shared) => shared.credentials)
-			: await Db.collections.Credentials.findOneByOrFail({ id: nodeCredential.id, type });
+		let credential: CredentialsEntity;
 
-		if (!credential) {
-			throw new Error(
-				`Credential with ID "${nodeCredential.id}" does not exist for type "${type}".`,
-			);
+		try {
+			credential = userId
+				? await Db.collections.SharedCredentials.findOneOrFail({
+						relations: ['credentials'],
+						where: { credentials: { id: nodeCredential.id, type }, userId },
+				  }).then((shared) => shared.credentials)
+				: await Db.collections.Credentials.findOneByOrFail({ id: nodeCredential.id, type });
+		} catch (error) {
```

Comment:
Thanks for fixing this. Maybe we should look for all uses of `OrFail` and make sure that they are all wrapped in a try-catch. 👍🏽